src/senaite/core/z3cform/widgets/duration/input.pt

```diff
@@ -0,0 +1,56 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      tal:omit-tag="">
+
+  <div tal:define="name       python:view.name;
+                   value      python:view.value;
+                   days       python:value.get('days', 0);
+                   hours      python:value.get('hours', 0);
+                   minutes    python:value.get('minutes', 0);
+                   seconds    python:value.get('seconds', 0);
+                   klass      python:view.klass;
+                   mode_klass python:'{}-input'.format(klass);
+                   klass      python:' '.join([klass, mode_klass]);"
+       tal:attributes="class klass">
+
+    <div class="form-row">
+      <div tal:define="name  string:${name}:days;"
+           tal:attributes="class python:'col-auto' if view.show_days else 'd-none'">
+        <label i18n:translate="duration_widget_label_days"
+               tal:attributes="for name">Days</label>
```

Comment:
From the docs:
The value of the for attribute must be a single id for a labelable form-related element in the same document as the `<label>` element. So, any given label element can be associated with only one form control.

Example:
```html
<label for="username">Click me to focus on the input field</label>
<input type="text" id="username" />
```

Reply:
Addressed with https://github.com/senaite/senaite.core/pull/2482/commits/ed195555fc5b9ce4955143cdbb7c7f72c5e72d40